tee-worker/ts-tests/integration-tests/common/utils/identity-helper.ts

```diff
@@ -49,15 +49,33 @@ export function parseIdGraph(
     return idGraph;
 }
 
-type TwitterValidationConfig =
+type Web2ValidationConfig =
     | {
+          identityType: 'Discord';
+          context: IntegrationTestContext;
+          signerIdentitity: CorePrimitivesIdentity;
+          linkIdentity: CorePrimitivesIdentity;
+          verificationType: 'PublicMessage';
```

Comment:
Would it work `verificationType: 'PublicMessage' | 'OAuth2';` ?

Reply:
yep 👍